config/routes.rb

```diff
@@ -1,10 +1,23 @@
 Rails.application.routes.draw do
+  resources :actors
   mount Ckeditor::Engine => '/ckeditor'
   resources :movies
   devise_for :users
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root 'pages#index'
-  get '/secret', to: 'pages#secret', as: :secret
-  get '/home', to: 'pages#home', as: :home
+  post 'movies/:movie_id/:actor_id/addActor', to: 'movies#addActor'
```

Comment:
Move it to resources :actors as a member route.
Use snake case naming convention